mappings/net/minecraft/util/collection/DefaultedList.mapping

```diff
@@ -7,7 +7,7 @@ CLASS net/minecraft/class_2371 net/minecraft/util/collection/DefaultedList
 	METHOD add (ILjava/lang/Object;)V
 		ARG 1 value
 		ARG 2 element
-	METHOD get (I)Ljava/lang/Object;
+	METHOD get getStack (I)Ljava/lang/Object;
```

Comment:
No. This is an override of a `List` method, and the name should **not** be touched. (There's also nothing about this class that's specific to slots.)